src/en/components/checkbox/base.md

```diff
@@ -1,14 +1,40 @@
 ---
+title: Checkbox
 layout: "layouts/base.njk"
 github: https://github.com/cds-snc/gcds-components/tree/main/src/components/gcds-checkbox
 figma: https://www.figma.com/community/file/1128687821123298228
 permalink: false
-tags: ['checkboxEN', 'header']
+tags: ["checkboxEN", "header"]
 ---
 
-# checkbox
+# {{ title }} <br>&lt;gcds-checkbox&gt;
 
-This element will have the content below it
+_Also called: checklist, check list._
+
+A checkbox is a question with a set of options for one or multiple selections.
 
 {% docLinks locale stage figma github %}
 {% enddocLinks %}
+
+<div class="b-sm text-secondary px-250 py-200 mt-400 mb-500">
```

Comment:
It seems that the `gcds-fieldset` gets wrapped in a `p` tag as well which adds `mb-400` to it.
Could you take a look and try to balance out the difference between the top and bottom spacing in the box?
Sorry didn't notice before that it was being wrapped in a `p`.

Reply:
Oh yeah, I must've missed that when I reverted the `mt-` to `mb-`